rs/registry/admin/bin/main.rs

```diff
@@ -2254,6 +2254,8 @@ impl ProposalPayload<AddNodeOperatorPayload> for ProposeToAddNodeOperatorCmd {
             .map(|s| parse_rewardable_nodes(s))
             .unwrap_or_default();
 
+        // TODO(DRE-583): Remove the deprecation notice after clients wrapping ic-admin
+        // manage to update their code to stop providing node_allowance.
```

Comment:
```suggestion
        // TODO(DRE-583): Remove the `--node-allowance` flag after callers of `ic-admin`
        // have migrated to `--max-rewardable-nodes`.
```

The important thing is not to get rid of the warning per se. The important thing is to stop pretending that we are doing what the user asked.